### DISCUSSION TITLE
Question about BOM & CPL files

### DISCUSSION THREAD

**Discussion by mwedmark** [frntc/Sidekick64#9] [General]
Hi I'm thinking of create a small batch of Sidekick64's. I see Gerber and I see BOM in text. To be able to simply order most of these presoldered I would need placement file for SMD soldering or the whole project and not just the gerber. It these files possible to get?

**Comment by hpingel** [comment]
Hi mwedmark, I'm just guessing here but I think that Sidekick64 is an evolving project that might see new PCB revisions. Handing out the information needed to replicate the PCB with components pre-soldered would send the signal that a final perfect PCB revision has been created and that the project is completely matured and not open for any future feature changes any more. Even if you only want to create a small batch for yourself, somebody else might want to create a bigger batch. Therefore I can understand that for a creative project like this it is better to prevent the option for industrial-scale reproduction of unlimited pre-soldered units.

**Reply by slesinger** [reply]
I have exactly the same request because
 - Sidekick64 is no longer available for purchase on Restore Store nor on Tindie. This would be my preferred way to go.
 - I can order plain PCB based on gerber files but because of my poor soldering skills it is a risk.
 
 That is why it makes sense to provide BOM and PCL files.
